recipes/llvmdev38/build.sh

```diff
@@ -0,0 +1,38 @@
+CMAKE_COMMON_VARIABLES=" -DCMAKE_INSTALL_PREFIX=$PREFIX/lib/llvm-3.8 \
```

Comment:
the binaries (llvm-config etc) will output paths to the libraries, probably best not to mess around with the structure after configure time